CMakeModules/FindCOLLADA.cmake

```diff
@@ -74,6 +74,7 @@ FIND_PATH(COLLADA_INCLUDE_DIR dae.h
     /opt/include
     /usr/freeware/include
     ${ACTUAL_3DPARTY_DIR}/include
+    ${ACTUAL_3DPARTY_DIR}/include/collada-dom2.5/
```

Comment:
There is a path suffix `collada-dom2.5` just below which should make it possible to find collada 2.5. No need to have this change.

Reply:
I'm pretty sure we're married to Collada DOM 2.4. 2.5 is really an alpha of the de-facto abandoned 3.x series, so has breaking changes.